<!-- {about} --><a name="about"></a>
  <h2 id="about" class="first">About <em>Bicyclus anynana</em></h2>
  <p>The squinting bush brown butterfly, <em>Bicyclus anynana</em>, is a member of the remarkably speciose Mycalesine butterflies that are distributed across the Old World tropics [1]. Males have a wingspan of about 35-40 mm, while females are slightly larger (45-49 mm) [2]. Adults are found primarily in woodland habitats across East Africa (from southern Sudan in the north to Swaziland in the south), and are typically observed flying close to the ground where they feed on fallen fruit [2]. The larvae of <em>B. anynana</em> feed on several species of grass. It is a model organism for the study of the development, plasticity and evolution of butterfly wing patterns [2,3,4]. In addition, <em>B. anynana</em> is one of the only insect species to exhibit seasonal <a href="http://en.wikipedia.org/wiki/Polyphenism">polyphenism</a>, whereby individuals that develop during the wet season differ in both behaviour and appearance to those that develop during the dry season [1,5]. Wet-season butterflies are smaller, have shorter lifespans, are more active, and show large and conspicuous eyespots on their wings in comparison to dry season individuals [6]. Thus, <em>B. anynana</em> is an important system through which to study the evolution of <a href="http://en.wikipedia.org/wiki/Phenotypic_plasticity">phenotypic plasticity</a>.</p>
  <p>Picture credit: <a href="http://europepmc.org/abstract/MED/20796293">Saenko <em>et al.</em> 2010. BMC Biol. 8:111-124</a>.</p>
  <p><!--nbsp--> <!--nbsp--></p>
  <p id="about"></p>
<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
  <h2 id="assembly" style="margin-bottom:5px">Assembly</h2>
	<p>The sequencing of the <em>B. anynana</em> genome was performed jointly by <a href="http://www.genomics.ed.ac.uk">Edinburgh Genomics</a>, the University of Edinburgh, Edinburgh, UK, and at the <a href="http://www.liv.ac.uk/genomic-research/">Centre for Genomic Research</a>, the University of Liverpool, Liverpool, UK, to an overall coverage of approximately 100x. These data were QC'd and assembled by the Blaxter Lab at the University of Edinburgh. Contigs were assembled using MaSuRCA, scaffolding and gap-filling was performed with SSPACE and GapFiller. The final assembly (denoted version 0.1) has 29,616 scaffolds (188,278 contigs), an N50 of 71,168 bp and a total span of approximately 553 Mb. The GC content is 38%.</p>
  <!--p>The <em>Bombyx mori</em> genome was assembled by the International Silkworm Genome Sequencing Consortium, a joint collaboration of the National Institute of Agrobiological Sciences (NIAS) in Japan and the Southwest University in Chongqing City, China. The consortium integrated two early assemblies into a single high-quality assembly with ~10x coverage of the genome [3]. Approximately 87% of the scaffolds are assembled into 28 linkage groups; this is not represented in Ensembl Metazoa, but is available at <a href="http://sgp.dna.affrc.go.jp/KAIKObase">KAIKObase</a> [4]. In release 22 of Ensembl Metazoa (March 2014) the assembly was updated to remove contigs (primarily bacterial contaminants and duplicate regions) that were not part of the canonical INSDC record.</p>
  <p>There are two sets of scaffold names in use for <em>Bombyx mori</em>. The INSDC records have scaffold names like "<em>:_Bm_scaf586_strain:_p50T_Dazao.</em>", but these have been abbreviated in Ensembl Metazoa (from release 22 onwards), to "<em>scaf586</em>", for example. (The longer INSDC nomenclature is listed as a synonym, however, so searches and uploads using that format should still go to the right place.) These scaffold numbers are also used by KAIKObase. Earlier Ensembl Metazoa releases used the nomenclature from SilkDB, which is formatted like "<em>nscaf29</em>"; note that the numbers in these names do not correpond to those in the INSDC names. The <a href="/tools.html">assembly converter tool</a> maps between our current assembly ("GCA_000151625.1") and the earlier version ("Bmor1").</p-->
<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
  <h2 id="annotation" style="margin-bottom:5px">Annotation</h2>
	<p><em>Ab initio</em> gene calling was performed with MAKER2, in conjuntion with transcriptomic datasets, and resulted in 14,133 gene models encoding 16,656 transcripts. These data can be veiwed and downloaded from <a href="http://bicyclus.org">bicyclus.org</a> (note that assembly version 0.1 corresponds to version 0.4 on <a href="http://bicyclus.org">bicyclus.org</a>).</p>
  <!--p>Aggregation of <em>ab intio</em> and similarity based gene predictions was achieved using a <a href="http://glean-gene.sourceforge.net">GLEAN</a>-like algorithm resulting in over 14,000 gene models. The annotations displayed in Ensembl Metazoa are imported from <a href="http://silkworm.genomics.org.cn">SilkDB</a> [5]. Changes to the scaffolds in release 22 of Ensembl Metazoa (see Assembly section) resulted in some genes mapping to new positions; in all but two cases the DNA sequence is unchanged. The exceptions are BGIBMGA009343, which lost 8 exons that were on a duplicated region, and BGIBMGA014289 which is truncated because part of the gene was on a contig believed to be a bacterial contaminant. Non-coding RNA genes were added using the <a href="http://metazoa.ensembl.org/info/genome/genebuild/ncrna.html">Ensembl Genomes pipeline</a>, and BLAST hits and protein features have been computed.</p-->
<!-- {annotation} -->
<!-- {references} -->
  <h2>References</h2>
  <ol>
    <li><a id="ref-01"></a><a href="http://www.ncbi.nlm.nih.gov/pubmed/21043782">Radiations of mycalesine butterflies and opening up their exploration of morphospace</a>.<br>Brakefield, PM. 2010. Am. Nat. 176 Suppl 1:S77-87.</li>
		<li><a id="ref-02"></a><a href="http://www.ncbi.nlm.nih.gov/pubmed/20147150">The African butterfly Bicyclus anynana: a model for evolutionary genetics and evolutionary developmental biology</a>.<br>Brakefield PM, Beldade P and Zwaan BJ. 2009. Cold Spring Harb Protoc. doi: 10.1101/pdb.emo122.</li>
    <li><a id="ref-03"></a><a href="http://www.ncbi.nlm.nih.gov/pubmed/12809139">Development, plasticity and evolution of butterfly eyespot patterns</a>.<br>Brakefield PM <em>et al</em>. 1996. Nature 348(6606):236-42.</li>
		<li><a id="ref-04"></a><a href="http://www.ncbi.nlm.nih.gov/pubmed/16713653">Evo-devo and constraints on selection</a>.<br>Brakefield PM. 2006. Trends Ecol Evol. 21(7):362-368.</li>
		<li><a id="ref-05"></a>The evolutionary significance of dry and wet season forms in some tropical butterflies.<br>Brakefield PM and Larsen TB. 1984. Biological Journal of the Linnean Society 22:1-12.</li>
		<li><a id="ref-06"></a><a href="http://www.ncbi.nlm.nih.gov/pubmed/17536166">Developmental plasticity and acclimation both contribute to adaptive responses to alternating seasons of plenty and of stress in <em>Bicyclus</em> butterflies</a>.<br>Brakefield PM, Pijpe J and Zwaan BJ. 2007. J Biosci. 32(3):465-475.</li>
	</ol>
<!-- {references} -->
    