<h3>Summary</h3>
    
<table class="ss tint species-stats">
  <tbody><tr class="bg2">
    <td class="data">Assembly:</td>
    <td class="value">Hmel1, INSDC Assembly <a href="http://www.ebi.ac.uk/ena/data/view/GCA_000313835.2">GCA_000313835.2</a>, December 2012</td>
  </tr>
  <tr>
    <td class="data">Database version:</td>
    <td class="value">80.1</td>
  </tr>
<tr class="bg2">
    <td class="data">Base Pairs:</td>
    <td class="value">269,658,870</td>
</tr><tr>
    <td class="data">Golden Path Length:</td>
    <td class="value">273,786,188</td>
</tr>
    <tr>
      <td class='data'>CEGMA Complete:</td>
      <td class='value'>83.06 %</td>
    </tr>
    <tr>
      <td class='data'>CEGMA Partial:</td>
      <td class='value'>94.35 %</td>
    </tr>
    <tr class="bg2"> <td class="data">Genebuild method:</td> <td class="value">Generated using MAKER</td> </tr></tbody></table><a href="/Heliconius_melpomene/Info/IPtop500?db=core">Table of top 500 InterPro hits</a><br><br>
    <h3>Gene counts</h3>
    <table class="ss tint species-stats">
  <tbody><tr class="bg2">
        <td class="data"><span class="glossary_mouseover">Coding genes</span>:</td>
        <td class="value">12,669</td>
        </tr>
      <tr>
        <td class="data"><span class="glossary_mouseover">Small non coding genes</span>:</td>
        <td class="value">3,534</td>
        </tr>
      <tr class="bg2">
        <td class="data"><span class="glossary_mouseover">Gene transcripts</span>:</td>
        <td class="value">16,363</td>
        </tr>
      
    </tbody></table>
  
    
  <h3>Coordinate Systems</h3>
<table class="ss tint species-stats">
    <tbody><tr class="bg2"> 
    <td class="data">scaffold</td>
    <td class="value">4309 sequences</td>
    </tr>
    <tr> 
    <td class="data">contig</td>
    <td class="value">11660 sequences</td>
    </tr></tbody></table>

    