<html>
<head>
<title>LepBase: Downloads</title>
</head>
<body>

<div id="static">

Coming soon...
<!--table class="blobs">
<tbody><tr>

<td class="threecol">
<h2 class="first"><a href="/info/data/export.html">Download a sequence or region</a></h2>
<div class="centered"><a href="/info/data/export.html"><img src="/img/download_sequence.gif" alt="" title="Find out more"></a></div>
<p>Click on the 'Export data' button in the lefthand menu of most pages to export:</p>
<ul>
<li>FASTA sequence</li>
<li>GTF or GFF features</li>
</ul>
<p>...and more!</p>
</td>



<td class="threecol">
<h2 class="first"><a href="/info/docs/api/">Fetch data programmatically</a></h2>
<div class="centered"><a href="/info/docs/api/"><img src="/img/download_api.gif" alt="" title="Find out more"></a></div>
<p>Write your own Perl scripts to retrieve small-to-medium datasets. All our data, as well as added functionality, is available 
through the Ensembl Perl API.</p>
<p>Use the API to retrieve gene and transcript sets, fetch alignments between sequences, compare allele frequencies and much more!</p>
<p>You can also use our <a href="http://rest.ensembl.org/">REST API</a> to retrieve data to process in the programming language of your choice.</p>
</td>


<td class="threecol">
<h2 class="first">Download <a href="/info/data/ftp/">databases</a> &amp; <a href="/info/docs/webcode/mirror/">software</a></h2>
<div class="centered"><a href="/info/"><img src="/img/download_code.gif" alt="" title="Find out more"></a></div>
<p>All of our data and software, including pipelines and web code, will be available free.</p>
<p>Meanwhile, take a look at some of data and code that LepBase is built on. 
<ul>
<li><a href="http://metazoa.ensembl.org/info/website/ftp/index.html">Download data via FTP</a></li>
<li><a href="https://github.com/Ensembl/ensembl-pipeline" rel="external">Ensembl pipeline on GitHub</a></li>
<li><a href="http://metazoa.ensembl.org/">Visit Ensembl Metazoa</a></li>
</ul>
</td>

</tr>
</tbody></table-->


</div>

</body>
</html>
