<html>
<head>
<title>LepBase: Source code</title>
</head>
<body>
           


<div id="static">



<h2>Source code</h2>
At Lepbase we support open source and open data.  We endeavour to make all the data we host available for <a href="download.html">bulk download</a> and our tools are powered by and released as open source software. 
<div style="max-width:1000px">
<table class="blobs">
  <tbody id="tbody">
    <tr>
      <td style="background-color:#cccccc;font-weight:bold;">
        The source code for all our projects is available on <span><a href="http://bitbucket.org"><img src="i/bitbucket-logo-long.png" title="bitbucket" height="16px"/></a></span> or <span><a href="http://github.com"><img src="i/github-logo-long.png" title="github" height="16px"/><img src="i/GitHub-Mark-32px.png" height="16px"/></a></span>
        <br/>
      </td>
    </tr>
    <tr>
      <td>
        Our modifications to ensembl-webcode for this site are available at <a href="https://bitbucket.org/lepbase/lepbase-ensembl"><span>lepbase-ensembl<img src="i/bitbucket-logo.png" height="16px"/></span></a>
        <br/>
      </td>
    </tr>
    <tr>
      <td>  
        Our data import pipeline (which includes search indexing) is available at  
        <a href="https://bitbucket.org/lepbase/lepbase-import"><span>lepbase-import<img src="i/bitbucket-logo.png" height="16px"/></span></a>
        <br/>
      </td>
    </tr>
    <tr>
      <td>
        We have written a generic GFF parser,
        <a href="https://github.com/rjchallis/gff_parser"><span>gff_parser<img src="i/GitHub-Mark-32px.png" height="16px"/></span></a>,
        which we use in our import pipeline to accommodate, validate and correct the diversity of GFF formats used in Lepidopteran genome projects
        <br/>
      </td>
    </tr>
    <tr>
      <td>
        Our fork of SequenceServer with changes to allow closer integration with our Ensembl instance is available at
        <a href="https://bitbucket.org/lepbase/lepbase-sequenceserver"><span>lepbase-sequenceserver<img src="i/bitbucket-logo.png" height="16px"/></span></a>
        <br/>
      </td>
    </tr>
    <tr>
      <td>
        Our assembly statistic visualisations are based on our 
        <a href="https://github.com/rjchallis/assembly_stats"><span>assembly_stats<img src="i/GitHub-Mark-32px.png" height="16px"/></span></a>
        javascript project
        <br/>
      </td>
    </tr>
    <tr>
      <td>
        We have written an API training environment,
        <a href="https://bitbucket.org/lepbase/lepbase-workshop"><span>lepbase-workshop<img src="i/bitbucket-logo.png" height="16px"/></span></a>,
        for use in training workshops, which we plan to make more widely applicable as an introduction to the Lepbase API
        <br/>
      </td>
    </tr>
    
  </tbody>
</table>
<table class="blobs">
  <tbody id="tbody">
    <tr>
      <td style="background-color:#cccccc;font-weight:bold;">
        Lepbase uses many other open source bioinformatics projects, including:
        <br/>
      </td>
    </tr>
    <tr>
      <td>
        The <a href="http://www.ebi.ac.uk/">EBI</a>'s <a href="http://www.ensembl.org/">Ensembl</a>/<a href="http://metazoa.ensembl.org/">EnsemblMetazoa</a> genome browser powers <a href="http://ensembl.lepbase.org">ensembl.lepbase.org</a>
        <br/>
        <br/>
        <a href="http://nar.oxfordjournals.org/content/43/D1/D662">Ensembl 2015</a>. Cunningham F, et al. 2015. Nucleic Acids Research 43:D662-D669
      </td>
    </tr>
    <tr>
      <td>
        <a href="http://blast.lepbase.org">blast.lepbase.org</a> is powered by the <a href="https://evolve.sbcs.qmul.ac.uk/wurm/">Wurm Lab</a>'s <a href="http://www.sequenceserver.org/">SequenceServer</a>
        <br/>
        <br/>
        SequenceServer: BLAST search made easy, Priyam A, Woodcroft BJ, Rai V, Wurm Y. in prep. http://sequenceserver.com
        <br/>
      </td>
    </tr>
    <tr>
      <td>
        Community annotations at <a href="http://webapollo.lepbase.org">webapollo.lepbase.org</a> are made possible by <a href="http://genomearchitect.org">Web Apollo</a>
        <br/>
        <br/>
        <a href="http://genomebiology.com/2013/14/8/R93/abstract">Web Apollo: a web-based genomic annotation editing platform</a>. Lee E, Helt GA, Reese JT, Munoz-Torres MC, Childers CP, Buels RM, Stein L, Holmes IH, Elsik CG, Lewis SE. 2013. Genome Biol 14:R93
        <br/>
      </td>
    </tr>
    
  </tbody>
</table>

</div>



</div>

</body>
</html>
